tests/integration/test_sso.py

```diff
@@ -1,23 +1,24 @@
 from sentry.models import AuthIdentity, AuthProvider
 from sentry.testutils import AuthProviderTestCase
-from sentry.testutils.silo import region_silo_test
+from sentry.testutils.silo import control_silo_test, exempt_from_silo_limits
 from sentry.utils.auth import SsoSession
 
 
-@region_silo_test
+@control_silo_test(stable=True)
```

Comment:
This test now runs twice -- once for monolith, and once for silo modes.  Nicely, the stubbing allows the same test setup to work in both modes while still enforcing data access boundaries.